src/LibraryManager.Vsix/Shared/LibraryHelpers.cs

```diff
@@ -48,13 +71,14 @@ public static bool IsSupported(this Project project)
             bool hasErrors = false;
             var telResult = new Dictionary<string, double>();
 
-            foreach (string configFilePath in configFilePaths)
+            foreach (KeyValuePair<string, Manifest> manifest in manifests)
```

Comment:
If we end up doing this in more places, we should probably make path a non-serialized field of Manifest.